.github/renovate.json5

```diff
@@ -157,6 +157,18 @@
       datasourceTemplate: 'docker',
       versioningTemplate: 'loose'
     },
+    {
+      customType: 'regex',
+      fileMatch: [
+        '^tests\\/utils\\/minio\\/minio\\.go$',
+      ],
+      matchStrings: [
+        'minioImage = "(?<depName>.+?):(?<currentValue>.*?)"',
+        'minioClientImage = "(?<depName>.+?):(?<currentValue>.*?)"',
+      ],
+      datasourceTemplate: 'docker',
+      versioningTemplate: "regex:^RELEASE\\.(?<major>\\d{4})-(?<minor>\\d{2})-(?<patch>\\d{2})T\\d{2}-\\d{2}-\\d{2}Z$"
+    },
```

Comment:
We should not create a custom regex for Minio; instead, we should utilize the generic support. Please refer to #7663 for more details on how it works.